gravitee-apim-console-webui/src/index.scss

```diff
@@ -96,6 +96,7 @@ body {
     display: flex;
     flex-direction: row;
     overflow: hidden;
+    height: 100%;
```

Comment:
I wonder if it's not related to the `height: 100vh;` L92

Reply:
not related...